pyproject.toml

```diff
@@ -48,6 +49,7 @@ disable = [
     "W0622", # Redefining built-in
     "R0903", # Too few public methods
 ]
+good-names = ["id"]
```

Comment:
What does `good-names` mean?

Reply:
They're good names, Zeke.

Reply:
(But seriously, this is to stop pylint from complaining about variables named `id`)